variants/WRAITH32_F051K6/readme.txt

```diff
@@ -0,0 +1,111 @@
+STM32F051K6 TIM Resources
+
+TIM1 : Advanced Control Timer, 16 Bit, 4 Channel
+TIM2 : General Timer, 32 Bit, 4 Channel
+TIM3 : General Timer, 16 Bit, 4 Channel
+TIM14 : General Timer, 16 Bit, 1 Channel
+TIM15 : General Timer, 16 Bit, 2 Channel
+TIM16 : General Timer, 16 Bit, 1 Channel
+TIM17 : General Timer, 16 Bit, 1 Channel
+TIM6 : Basic Timer, 16 Bit
+
+// --------------------------------------------------------------------------
+
+TIM1 : Motor Control Timer
+TIM14 : Motor Commutation Timer
+TIM15 : RC Input Measurement
+TIM16 : System Millisecond Timer
+
+Input : PA2, TIM15_CH1
+
+LED_RED : PA15
+LED_GREEN : PB3
+LED_BLUE : PB4
+
+A_FET_HI : PA10
```

Comment:
All of these defines could be added in `variant.h` for convenience it you want.

Reply:
Done

Reply:
I do not see any define added in variant.h. Anyway this is not an issue. I would only raise that it can be added 😉